src/polymerday-sponsors-list.html

```diff
@@ -1,32 +1,72 @@
 <link rel="import" href="../bower_components/polymer/polymer.html">
-
 <link rel="import" href="polymerday-data.html">
+<link rel="import" href="polymerday-sponsor-item.html">
 
 <dom-module id="polymerday-sponsors-list">
-
   <template>
 
-  <style>
-
-    a {
-      text-decoration: none;
-    }
-  </style>
-
-  <polymerday-data
-    page="sponsors"
-    subpage="{{sponsorsType}}"
-    data="{{sponsors}}"></polymerday-data>
-
-  <template is="dom-if" if="[[sponsors.length]]">
-    <polymerday-section-title title="[[sponsorsTitle]]"></polymerday-section-title>
-    <template is="dom-repeat" items="{{sponsors}}" as="sponsor">
-        <a href="[[sponsor.link]]">
-          <img src="[[sponsor.logo]]" alt="[[sponsor.name]]"/>
-        </a>
-    </template>
-  </template>
+    <style>
+
+      .sponsors--container {
+        display: flex;
+        flex-flow: row wrap;
+        justify-content: flex-start;
+      }
+
+      .sponsor-platinum {
```

Comment:
Platinum, gold, iron and paper could change and this stop working. Use a property.